device/dhcp/clients/v6/dhcpv6client.yaml

```diff
@@ -0,0 +1,146 @@
+      type: object
+      x-status:
+        status: under_review
+      required: [name]
+      properties:
+        name:
+          x-include: ../../../../common/common.yaml#/components/schemas/Named.Object/properties/name
+          x-field-uid: 1
+        rapid_commit:
+          description: >-
+            If Rapid Commit is set, client initiates Rapid Commit two-message exchange by including Rapid Commit option 
+            in Solicit message.
+          type: boolean
+          default: false
+          x-field-uid: 2
+        ia_type:
+          $ref: '#/components/schemas/Device.Dhcpv6client.IaType'
+          x-field-uid: 3
+        duid_type:
+          $ref: '#/components/schemas/Device.Dhcpv6client.DuidType'
+          x-field-uid: 4
+
+    Device.Dhcpv6client.IaType:
+      type: object
+      properties:
+        choice:
+          description: >-
+            Identity Association: a collection of leases assigned to a client. Each IA has an associated IAID. 
+            A client may have more than one IA assigned to it, one for each of its interfaces. Each IA holds 
+            one type of lease, like an identity association for temporary addresses (IA_TA) holds temporary addresses, 
+            and an identity association for prefix delegation (IA_PD).
+          type: string
+          default: iana
+          x-field-uid: 1
+          x-enum:
+            iana:
+              x-field-uid: 1
+            iata:
+              x-field-uid: 2
+            iapd:
+              x-field-uid: 3
+            ianapd:
+              x-field-uid: 4
+        iana:
+          $ref: '#/components/schemas/Device.Dhcpv6client.IaTimeValue'
+          x-field-uid: 2
+        iata:
+          $ref: '#/components/schemas/Device.Dhcpv6client.NoIaTimeValue'
+          x-field-uid: 3  
+        iapd:
+          $ref: '#/components/schemas/Device.Dhcpv6client.IaTimeValue'
+          x-field-uid: 4
+        ianapd:
+          $ref: '#/components/schemas/Device.Dhcpv6client.IaTimeValue'
+          x-field-uid: 5
+
+    Device.Dhcpv6client.IaTimeValue:
+      description: >-
+        The container for the suggested times at which the client contacts the server or any available server.
+      type: object
+      properties:
+        ta_1:
+          description: >-
+            The suggested time at which the client contacts the server from which the addresses were obtained to 
+            extend the lifetimes of the addresses assigned.
+          type: integer
+          format: uint64
+          maximum: 2147483647
+          default: 302400
+          x-field-uid: 1
+        ta_2:
+          description: >-
+            The suggested time at which the client contacts any available server to extend the lifetimes of the addresses assigned.
+          type: integer
+          format: uint64
+          maximum: 2147483647
+          default: 483840
+          x-field-uid: 2
+
+    Device.Dhcpv6client.NoIaTimeValue:
```

Comment:
This looks like is an empty object , we can probably remove it and describe the types properly in the enum list itself.